traits/experimental/servicetrait/config/samples/statefulsetworkload/sample_application_config.yaml

```diff
@@ -17,4 +17,4 @@ spec:
             metadata:
               name:  example-appconfig-trait
```

Comment:
I think we can't specify metadata.name  in trait. Trait was not a instance in spec, it become a instance at runtime. So we can automatically fill this field when running a trait instance